shippy-node-example/shippy-app/js/shippy-storage.js

```diff
@@ -0,0 +1,162 @@
+/**
+ * The SessionStorage controller used by Shippy
+ *
+ * Exposes the interface specified as return statement to the module at the bottom.
+ */
+Shippy.Storage = (function () {
+
+	/*
+	 * This MIME types were extracted from:
+	 * https://developer.mozilla.org/en-US/docs/Web/HTTP/Basics_of_HTTP/MIME_types/Complete_list_of_MIME_types
+	 * However, this is not a comprehensive list
+	 */
+	var extToMimes = {
+		'img': 'image/jpeg',
+		'png': 'image/png',
+		'css': 'text/css',
+		'gif': 'image/gif',
+		'htm': 'text/html',
+		'html': 'text/html',
+		'ico': 'image/x-icon',
+		'jpeg': 'image/jpeg',
+		'jpg': 'image/jpeg',
+		'js': 'application/javascript',
+		'json': 'application/json',
+		'xml': 'application/xml',
+		'default': 'text/plain',
+		'woff': 'font/woff',
+		'woff2': 'font/woff2',
+		'ttf': 'font/ttf'
+	};
+
+	// Converts a file to a MIME type
+	function fileToMime(file) {
+		var index = file.lastIndexOf('.');
+		if (index !== -1) {
+			var extension = file.substring(index + 1, file.length);
```

Comment:
You just want the file extension? Maybe this is easier: `file.split(".").pop()`?

Reply:
fixed